### PR TITLE
alpscore : update to version 0.5.4

### PR DESCRIPTION
**Pull request by aeantipov**
The ALPSCore project, based on the [ALPS](http://alps.comp-phys.org/mediawiki/index.php/Main_Page) (Algorithms and Libraries for Physics Simulations) project, provides generic algorithms and utilities for physics problems. It strives to increase software reuse in the physics community.

This release: 
  1. Incorporated a library to operate Green's Functions;
  1. Reworked static/dynamic build configuration;
  1. Bugfixes and miscellaneous improvements.

For copyright see [COPYRIGHT.TXT](https://github.com/ALPSCore/ALPSCore/blob/v0.5.2/COPYRIGHT.TXT). For licensing see [LICENSE.TXT](https://github.com/ALPSCore/ALPSCore/blob/v0.5.2/LICENSE.TXT). For acknowledgment in scientific publications see [ACKNOWLEDGE.TXT](https://github.com/ALPSCore/ALPSCore/blob/v0.5.2/ACKNOWLEDGE.TXT).

### CODE REVIEW

**Comment by aeantipov** [comment]
Hey, just wanted to check on progress here.

File: alpscore.rb [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
   head "https://github.com/ALPSCore/ALPSCore.git"
 
   bottle do
     cellar :any
-    sha256 "933fee39d392c6de4176244ea92e5f4937157fa01893ba643a29bded50e6caaf" => :el_capitan
-    sha256 "e679dcc5dd2064daec7cd0c87e99dde14cdbf1a2aeb368503e093d2f460c1008" => :yosemite
-    sha256 "c4341da2fc657191965a11dc0cb06e8ca151399e6dbed5c7fc5cbc0cfacf2d49" => :mavericks
+    sha256 "ca3718cf06979245642bf7480b4b0dbdb756ed7b0f02885fd37a3f9b2bccae8c" => :el_capitan
+    sha256 "5eabcd012359f7b817e880c84ddd8b6e6ed71e449a6c8c241bf035efc9ecf4d6" => :yosemite
+    sha256 "7429a356163589a99f123f3200dfd0e234cdf46a83827fa3c24f767a0c1e99a9" => :mavericks
```
Comment (dpo) [inline alpscore.rb]: Don't change the `bottle` section (as requested in the PR template...)

Comment (aeantipov) [inline alpscore.rb]: Oh, sorry about that - haven't noticed it.

**Comment by aeantipov** [comment]
Sorry about that, fixed now

**Pull request closed without merging**